packages/env/lib/config.js

```diff
@@ -96,6 +98,8 @@ module.exports = {
 				core: null,
 				plugins: [],
 				themes: [],
+				portNumber: 8888,
+				testsPortNumber: 8889,
```

Comment:
I think let's just use `port` and `testsPort` as the names for the `.wp-env.json` fields and related variables. It's shorter, we already validate the type, and most folks know that a port is a number.

Reply:
Done!